Cargo.toml

```diff
@@ -38,3 +38,7 @@ rhai = { version = "1.15.1", features = ["serde", "sync"] }
 once_cell = "1.18.0"
 base64 = "0.21.2"
 anyhow = "1.0.75"
+
+[features]
+default = []
```

Comment:
Should it rather have a `spiritnet` feature, with the `default` feature not using either of them by default? Or even removing the `default` feature for now, and only explicitly have `spiritnet`  and `peregrine` features?

Reply:
having one feature to activate "test environment" mode is simpler in my opinion.

Reply:
The change would reflect the logic in the Docker image naming. I am also fine with this. Your call as to whether accept or ignore my suggestion 👍

Reply:
Can you tell me how to do mutually exclusive feature flags? I know I could have something like "if(spiritnet) and not peregrine" but this wouldnt prevent the user trying to build something with spiritnet and peregrine enabled. If we just have one feature flag its easier. I would be honestly interested to know a good pattern to build this, but wasnt able to find something online.